Sources/Widgets/Widgets3D/ResliceCursorWidget/index.js

```diff
@@ -378,6 +380,23 @@ function vtkResliceCursorWidget(publicAPI, model) {
 
     renderer.getActiveCamera().setFocalPoint(...estimatedFocalPoint);
     renderer.getActiveCamera().setPosition(...estimatedCameraPosition);
+
+    if (!resetViewUp) {
+      // Compute the new view up by projecting vector [0, 1] in 3D
+      const view = renderer.getRenderWindow().getViews()[0];
+      const dims = view.getViewportSize(renderer);
+      const aspect = dims[0] / dims[1];
+
+      const p0 = [0, 0, 0];
+      const p1 = [0, 1, 0];
+
+      const worldP0 = renderer.normalizedDisplayToWorld(...p0, aspect);
+      const worldP1 = renderer.normalizedDisplayToWorld(...p1, aspect);
+      const newViewUp = vtkMath.subtract(worldP1, worldP0, [0, 0, 0]);
+      vtkMath.normalize(newViewUp);
+
+      viewUpFromViewType[viewType] = newViewUp;
+    }
```

Comment:
shouldn't we do that instead in `updateReslicePlane`. 
That way `resetCamera` always apply `viewUpFromViewType`

Btw, `viewUpFromViewType` should probably be stored in the widget state.

Reply:
`updateReslicePlane` doesn't have access to the renderer (and so the camera). I'm afraid it will overload the method for nothing.

Reply:
You wouldn't need access to the camera in `updateReslicePlane`.
Indeed the camera view up is always sync-ed with `viewUpFromViewType`. Therefore `updateReslicePlane` can use the `viewUpFromViewType` value to know the current camera view-up value. No ?

Reply:
I'm not sure to understand why the `updateReslicePlane` would know the current camera view up ?
Moreover, the first time we initialize the widget, we have to compute the view up as it's computed in `updateReslicePlane`. Then, if the user doesn't want to reset the viewup, it will be computed in the `resetCamera`

Reply:
`updateReslicePlane` knows the current camera view up because it is the one that sets the view up (`updateReslicePlane` sets `viewUpFromViewType` and `resetCamera` sets the camera view up from `viewUpFromViewType`).

Reply:
We need to align on this. I don't understand what you want to do.